markdown/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -1,339 +1,223 @@
+| `--tunnel`                   |     | Create a tunnel URL which points to your local server which anyone can use            |
+| `--theme-editor`             |`-e` | Run Theme Editor server                                                               |
+| `--no-cache`                 |`-n` | Turns off caching for API resource data (cache refreshes every 5 minutes)             |
+| `--theme-editor-port [port]` |     |Run the Theme Editor on a different port                                               |
+| `--help`                     |`-h` | output usage information                                                              |
 
 <div class="HubBlock--callout">
 <div class="CalloutBlock--warning">
 <div class="HubBlock-content">
 
 <!-- theme: warning -->
 
-### Troubleshooting Token Error
-> If you receive the following error message, `Unauthorized, please use a valid username/token`, then store token authentication has failed. In this case, please evisit [Authorizing and Initializing the Stencil CLI](/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands).
+### Authentication Errors
+> If you receive an `Unauthorized, please use a valid username/token` error, authentication has failed. Triple check the API token supplied is correct. For more information on creating store API accounts and generating tokens, see: [Obtaining Store API Credentials](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#step-3-serve-live-preview). 
 
 </div>
 </div>
 </div>
 
 ---
 
-<a href='#stencil_local-launch-theme-and-editor' aria-hidden='true' class='block-anchor'  id='stencil_local-launch-theme-and-editor'><i aria-hidden='true' class='linkify icon'></i></a>
+<a id="stencil-bundle"></a>
 
-## Local Launch: Theme and Theme Editor
+## `stencil bundle`
 
-If you want to view or reconfigure the Theme Editor/Store Design settings, you can run Stencil locally while also launching a local version of Theme Editor. This will help you see the UX changes that result from editing the `schema.json` file.
+Bundles up the theme into a structured `.zip` file, which can be uploaded to BigCommerce.
 
-To run Stencil locally while also launching a local version of Theme Editor, start in your theme directory and run the same command as above (`stencil start`), but append the `-e` option:
+**Usage:**
 
-`stencil start -e`
+```shell
+stencil bundle
+```
 
-The Theme Editor server will run in parallel with the stencil theme at https://localhost:8181.
+---
 
-### Customizing Theme Editor's Port
+<a id="stencil-push"></a>
 
-You can override Theme Editor's default port by appending the "--theme-editor-port [port]" option. For example, to specify port `9000`, you would enter:
+## `stencil push`
 
-`stencil start -e --theme-editor-port 9000
-`
+Bundles up the theme into a structured `.zip file`; then directly uploads (pushes) the `.zip` to BigCommerce.

```

Comment:
Bundles the theme into a structured `.zip file`; then directly uploads (pushes) the `.zip` to BigCommerce.